src/main/resources/org/jenkinsci/plugins/ghprb/GhprbTrigger/global.groovy

```diff
@@ -46,6 +46,9 @@ f.section(title: descriptor.displayName) {
     f.entry(field: "cron", title: _("Crontab line"), help: "/descriptor/hudson.triggers.TimerTrigger/help/spec") {
       f.textbox(default: "H/5 * * * *", checkUrl: "'descriptorByName/hudson.triggers.TimerTrigger/checkSpec?value=' + encodeURIComponent(this.value)") 
     }
+    f.entry(field: "blackListCommitAuthor", title: _("Blacklist commit authors")) {
```

Comment:
Could you add a bit more, pointing out that this will be overridden if a job's config is using blacklist commit authors?

Maybe something like:
`Blacklist commit authors. (Note: this can be overridden by job-specific configuration.)